ntoskrnl/include/internal/mm.h

```diff
@@ -269,6 +268,11 @@ typedef struct _MEMORY_AREA
     } SectionData;
 } MEMORY_AREA, *PMEMORY_AREA;
 
+#define MI_SET_MEMORY_AREA_VAD(Vad) do { (Vad)->u.VadFlags.Spare |= 1; } while (0)
```

Comment:
Looks like you could more simply do: `((Vad)->u.VadFlags.Spare |= 1)` .

Reply:
This is common practice.